tests/TestDataService.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '../src/index';
+
+test('Data Service', async ({
+    TestDataService,
+    DefaultSalesChannel,
+}) => {
+
+    const category = await TestDataService.createCategory(DefaultSalesChannel.salesChannel.navigationCategoryId, { name: 'Custom Category' });
+    expect(category.name).toEqual('Custom Category');
+
+    const product = await TestDataService.createProductWithImage({ description: 'Test Description' });
+    expect(product.description).toEqual('Test Description');
+    expect(product.coverId).toBeDefined();
+
+    const customer = await TestDataService.createCustomer(DefaultSalesChannel.salesChannel, 'mr', { firstName: 'Luke', lastName: 'Skywalker' })
```

Comment:
Same thing with defaults and I'd also prefer props if all of them are optional

Reply:
I added more defaults and put the overrides to the front. I'm not a big fan of parameters as objects if it is just a few parameters.